CHANGELOG.md

```diff
@@ -30,6 +31,39 @@
 - [1.5.0](#150)
 - [1.0.0](#100)
 
+# 2.6.1
+
+This release mainly bugfix and improves test cases for `Manager API`.
```

Comment:
```suggestion
This release mainly contains several bugfixes and test case improvements for `Manager API`.
```

Reply:
By the way, this sentence is not correct as there are some features listed in the change log context.